locales/de/translation.json

```diff
@@ -2482,17 +2482,19 @@
       "details": {
         "earn": {
           "titleGasSubsidy": "Maximiere Verdienste ohne Gasgebühren*",
-          "title": "Maximiere Verdienste",
-          "subtitle": "Verdiene an deinen Stablecoins und erhalte Rewards, wenn du einen Pool belieferst.",
+          "title": "Pools erkunden und vergleichen",
+          "subtitle": "Vergleiche Verdienstmöglichkeiten und Risiken. Tausche, kaufe oder transferiere Token, um dem besten Pool für dich beizutreten.",
           "footnoteSubsidy": "*Gasgebühren werden vorübergehend von Valora übernommen"
         },
         "manage": {
           "title": "Direkt von Valora aus verwalten",
-          "subtitle": "Wir kümmern uns um die Einrichtung, damit du deine Position direkt von Valora aus verwalten kannst."
+          "titleV1_92": "Verwalte direkt von {{appName}}aus",
```

Comment:
@mpgaarciaa this translation `Verwalte direkt von {{appName}}aus` looks a bit off.

Reply:
After talking with @mpgaarciaa this can happen with auto-generated German translations and is fine to merge.